### ISSUE TITLE
multiple schema documents

### ISSUE THREAD

**Issue by karenetheridge** [karenetheridge/JSON-Schema-Draft201909#7]
- must be able to provide multiple schema documents to the evaluator
- they need not be loadable externally (e.g. disk, network)
- they must have URIs associated with them
- they must be indexed somehow so $refs can be used against them (either literally or with a fragment attached)

https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.8.2.4.5

**Comment by karenetheridge** [comment]
These should all be supported, whether or not loading from the network is implemented/enabled:

```
$js->evaluate($data, $document_object);
$js->evaluate($data, 'https://json-schema.org/draft/2019-09/schema'); # load from internal cache
$js->evaluate($data, 'https://foo.bar.com/schema.json');                    # requires $js->add_schema(...) first
$js->evaluate($data, 'https://foo.bar.com/schema.json#/$defs/foo');  # ""
$js->evaluate($data, 'https://foo.bar.com/schema.json#some_id');       # ""
```

**Comment by karenetheridge** [comment]
Somewhat implemented in version 0.004 (commit 5c520e6ae6ce3c92ddc55cd034aa8d02d40eb559), but there is no ability yet to manually add additional documents.

**Comment by karenetheridge** [comment]
This is now done in version 0.006.
